doc/source/first-time-contributors.rst

```diff
@@ -0,0 +1,314 @@
+First time contributors
+=======================
+
+This guide is geared towards people that want to get involved in Flower
+but that are not used to contributing on GitHub projects.
+
+If GitHub contributions have no secret for you, 
+you can directly checkout our 
+`getting started guide for contributors <https://flower.dev/docs/getting-started-for-contributors.html>`_ 
+and examples of `good first contributions <https://flower.dev/docs/good-first-contributions.html>`_.
+
+Setting up the repository
+-------------------------
+
+1. **Create a GitHub account and setup Git**
+    Git is a distributed version control tool. This allows for an entire codebase's history to be stored and every developper's machine.
+    It is a software that will need to be installed on your local machine, you can follow this `guide <https://docs.github.com/en/get-started/quickstart/set-up-git>`_ to set it up.
+
+    GitHub, itself, is a code hosting platform for version control and collaboration. It allows for everyone to collaborate and work from anywhere on remote repositories.
+
+    If you haven't already, you will need to create an account on `GitHub <https://github.com/signup>`_. 
+
+    The idea behind the generic Git and GitHub workflow boils down to this: 
+    you download code from a remote repository on GitHub, make changes locally and keep track of them using Git and then you upload your new history back to GitHub.
+
+2. **Forking the Flower repository**
+    A fork is a personnal copy of a GitHub repository. To create one for Flower, you must navigate to https://github.com/adap/flower (while connected to your GitHub account)
```

Comment:
```suggestion
    A fork is a personal copy of a GitHub repository. To create one for Flower, you must navigate to https://github.com/adap/flower (while connected to your GitHub account)
```

Reply:
Does the URL get rendered in a clickable way?

Reply:
Yes it does